vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/scripts/new_accel_brake_map_server.py

```diff
@@ -0,0 +1,339 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
[nits] I think this is unnecessary anymore.

```suggestion
```

Reply:
Deleted the line